### PR TITLE
Work branch

### PR DESCRIPTION
**Pull request by EyasuTeshome**
To finish the capstone project, HTML, CSS, and Javascript files are included. I did the project assuming it could be used for a concert. I have made it responsive for different screen sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    <section class="performers">
+        <h3 class="title-speakers">Featured Speakers</h3>
+        <hr class="h3-hr">
+        <div class="flex-12">
+        <div class="performers1"></div>
+      </div>
+
+    </section>
+
```
Comment (Lameck1) [inline index.html:137]: - [ ] Immediately after the `Featured Speakers` section, there are two sections that have left out in the implementation of the provided design 👇🏼:

  image
  
  The `partner` and the `footer` sections displayed in the image above need to be implemented. Please work on this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    </section>
+    <section class="main-program">
+        <p class="main-program-p">Main Program</p>
+        <hr class="h3-hr">
+        <div class="program-container">
+        <div class="program program1">
+            <img class="program-item" src="./img/opening.png" alt="">
+            <h3 class="program-item h3-program">Opening</h3>
+            <p class="program-item">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Natus necessitatibus commodi at! Rem, accusamus reprehenderit.</p>
+        </div>
```
Comment (Lameck1) [inline index.html]: - [ ] I suggest we refactor the cards in this section, just a little bit. Please consider the font size and the width and height of each card. They should be similar if not exact. The implementation currently looks a little bit large. You may do the comparison below:

  | Design | The Implementation |
  | --- | --- |
  | image | image |
  
- [ ] Please consider working on the background of the `Main Program` Section. As evident above, the backgrounds doesn't seem to match or be somehow similar.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            <a class="join-button" href="#">Join Zema Songs Concert 2022</a>
+            <a class="seeall-link" href="">SEE The WHOLE PROGRAM</a>
+        </div>
+    </section>
+
+    <section class="performers">
+        <h3 class="title-speakers">Featured Speakers</h3>
+        <hr class="h3-hr">
+        <div class="flex-12">
+        <div class="performers1"></div>
```
Comment (Lameck1) [inline index.html:116]: - [ ] In the mobile view. There is a design implementation that we still have not considered yet. Take a look 👇🏼:

  | Design | Implementation |
  | --- | --- |
  | Screenshot 2022-06-30 193900 | image |
  
  As can be realized above, the design of this section in the mobile view only displays two speakers. The rest can be viewed by clicking on the button provided just after the two speakers.
  
  - [ ] Please refactor the implementation of this section's design.
  - [ ] Please add the `MORE` button. It should only be visible in the mobile view.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @EyasuTeshome, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
- [ ] Please let us fulfill the following requirements, they are part of project review.

  **ONE**
  > - Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a **maximum length of 5 minutes**. The content of the video should include:
    >   - A description of the project.
    >   - A demo of the project features (different pages and different screen sizes).
    >   - You should also highlight some interesting piece of code or something you built that you are very proud of.
    > - For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.    
    > - **Add the video link to your pull request description.**
    
    **TWO**
    > - Deploy the project and check to make sure everything works as planned. The live version that is currently available in the README is the one for your portfolio not this project.
    
    **THREE**
    > - The README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

- [ ] _Check the comments under the review too._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (EyasuTeshome) [reply index.html:116]: @Lameck1 please this is optional to add the button it's not required on the project

**Comment by EyasuTeshome** [comment]
I have fixed all the comments given by my code reviewer. The "More" button after the featured speakers is optional in the project description. I will do that one after course time as mentioned in the requirement "Any exploration here should be done outside program time."

File: index.html (hunk 3 above)
Comment (Lameck1) [reply index.html:116]: @EyasuTeshome, my bad. You are totally right, I beg your pardon for not realizing this.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @EyasuTeshome, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**